packages/at_secondary_server/lib/src/server/at_secondary_config.dart

```diff
@@ -61,9 +61,10 @@ class AtSecondaryConfig {
   static final int _runRefreshJobHour = 3;
 
   //Connection
-  static final int _inboundMaxLimit = 10;
-  static final int _outboundMaxLimit = 10;
-  static final int _inboundIdleTimeMillis = 600000;
+  static final int _inboundMaxLimit = 200;
```

Comment:
I know there's a bit of following suit here with the existing code, but could (should?) these be `const` rather than `static final`?

Reply:
> All looks good...
> 
> My only thought was to document how we roll back to normal while its fresh in your mind now. So when the time comes its easy, for some one else to do. Plus set up a remind ourselves to roll back in the calendar or something so things do get migrated back when the clients are all updated..

Sure thing. I don't see any real need to revert later but if we do then it's a change to config.yaml where I've documented these config parameters well.

I will create a ticket to review in a couple of months.

Reply:
> I know there's a bit of following suit here with the existing code, but could (should?) these be `const` rather than `static final`?

👍 Will change